nearcore/src/lib.rs

```diff
@@ -360,21 +360,15 @@ pub fn start_with_config_and_synchronization(
     );
     let snapshot_callbacks = SnapshotCallbacks { make_snapshot_callback, delete_snapshot_callback };
 
-    let (partial_witness_actor, partial_witness_arbiter) = if config.validator_signer.is_some() {
-        let my_signer = config.validator_signer.clone().unwrap();
-        let (partial_witness_actor, partial_witness_arbiter) =
-            spawn_actix_actor(PartialWitnessActor::new(
-                Clock::real(),
-                network_adapter.as_multi_sender(),
-                client_adapter_for_partial_witness_actor.as_multi_sender(),
-                my_signer,
-                epoch_manager.clone(),
-                storage.get_hot_store(),
-            ));
-        (Some(partial_witness_actor), Some(partial_witness_arbiter))
-    } else {
-        (None, None)
-    };
+    let (partial_witness_actor, partial_witness_arbiter) =
+        spawn_actix_actor(PartialWitnessActor::new(
+            Clock::real(),
+            network_adapter.as_multi_sender(),
+            client_adapter_for_partial_witness_actor.as_multi_sender(),
+            config.validator_signer.clone(),
```

Comment:
Nice, so now you always spawn this actor but it's only going to do something when the signer is set right?

Reply:
Yes